config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -266,42 +305,45 @@ processors:
             
       - metric_name: system.swap.paging_ops
         action: update
-        new_name: agent.googleapis.com/swap/io
+        new_name: swap/io
         operations:
           # delete singular type dimension, retaining only direction
           - action: aggregate_labels
             label_set: [ direction ]
             aggregation_type: sum
 
 exporters:
-  stackdriver:  
+  stackdriver/agent:  
+    project:  # stackdriver will use the default GCE credentials
```

Comment:
I don't think that line is needed?

Reply:
Not 100% sure, but I'll remove. I can't really test anymore since I don't have any GCP credentials, but from the documentation seems like this should be OK. https://pkg.go.dev/contrib.go.opencensus.io/exporter/stackdriver/?tab=doc#Options

Reply:
I've never added it and it worked fine for me